src/sql/workbench/contrib/notebook/test/browser/notebookMarkdown.test.ts

```diff
@@ -56,7 +57,9 @@ suite('NotebookMarkdownRenderer', () => {
 		notebookMarkdownRenderer.setNotebookURI(URI.parse('maddy/temp/file1.txt'));
 		let result: HTMLElement = notebookMarkdownRenderer.renderMarkdown({ value: `[Link to relative path](../test/.build/someimageurl)`, isTrusted: true });
 		if (process.platform === 'win32') {
-			assert.strictEqual(result.innerHTML, `<p><a href="\\maddy\\test\\.build\\someimageurl" data-href="\\maddy\\test\\.build\\someimageurl" title="\\maddy\\test\\.build\\someimageurl">Link to relative path</a></p>`);
+			let fullPath = path.resolve('.');
+			let diskDrive = fullPath.substring(0, fullPath.indexOf(':') + 1);
+			assert.strictEqual(result.innerHTML, `<p><a href="${diskDrive}\\maddy\\test\\.build\\someimageurl" data-href="${diskDrive}\\maddy\\test\\.build\\someimageurl" title="${diskDrive}\\maddy\\test\\.build\\someimageurl">Link to relative path</a></p>`);
```

Comment:
Are we breaking network paths?

Reply:
nit: while you're here - I would suggest just using some generic term such as foo instead of our names.

Reply:
No UNC paths work properly for linking absolute and relative paths via WYSIWYG.

Reply:
Since this test does not have to do with unc paths reverted to previous but added test regarding the marked js issue.